pkg/hashivault/manager.go

```diff
@@ -56,6 +56,27 @@ func (m *manager) GetSecret(ctx context.Context, path string) (EvergreenSecretsF
 	return es.get, nil
 }
 
+// Wrapper around GetSecret that gets only a single secret at the given key.
```

Comment:
Kom på - kommentarer på funksjoner i Go burde alltid starte med navnet til funksjonen, så denne burde endres til `// GetStaticSecretAtKey wraps the GetSecret...`

Reply:
Fikser her: https://github.com/3lvia/libraries-go/pull/46